performance/include/traccc/resolution/fitting_performance_writer.hpp

```diff
@@ -1,6 +1,6 @@
 /** TRACCC library, part of the ACTS project (R&D line)
  *
- * (c) 2022 CERN for the benefit of the ACTS project
+ * (c) 2022-2024 CERN for the benefit of the ACTS project
```

Comment:
BTW could consider following ACTS' change to only list the initial year instead of updating files.

Reply:
I'm actually partial to the convention we have in the ATLAS offline code, where we show the last modification date of each file. 🤔

Reply:
Sure. We had a discussion about this, and ultimately decided not to duplicate modification date (that's what blame is for) since it avoids it getting out of date, and the actual year being legally meaningless anyway.

Reply:
Yeah strong support for the ACTS style of not updating these; avoids a lot of pain in the rear and it has zero legal meaning.

Reply:
You guys are making it sound like it's a black and white issue. But it's very much a shade of grey in my mind.

For instance, what about when we re-design some code. Let's say we break up a class into two separate classes, with "new" names. What would be the appropriate date on the files of the new classes? Would one use the current year? The year when the original class was written?

In the scheme that we've been using so far in the R&D projects, the new files would reflect both that their code came from an earlier time, and that that code has now moved into a new place.

Paul is correct, even we discussed about this in the past. There can be different legitimate conventions for this. But given that the convention in this repository has **not** been the one lobbied for now, if we want to switch to that convention, we need to do it clearly.

I.e. somebody should open a PR, and update all files accordingly. 🤔 And update the contribution guide. (I'm just kidding. I meant **write** a contribution guide...)